mantle/cmd/kola/qemuexec.go

```diff
@@ -71,6 +72,7 @@ func init() {
 	cmdQemuExec.Flags().StringSliceVar(&ignitionFragments, "add-ignition", nil, "Append well-known Ignition fragment: [\"autologin\"]")
 	cmdQemuExec.Flags().StringVarP(&hostname, "hostname", "", "", "Set hostname via DHCP")
 	cmdQemuExec.Flags().IntVarP(&memory, "memory", "m", 0, "Memory in MB")
+	cmdQemuExec.Flags().StringArrayVarP(&addDisks, "add-disks", "D", []string{}, "Additional disks, human readable size per entry")
```

Comment:
How about

```suggestion
	cmdQemuExec.Flags().StringArrayVarP(&addDisks, "add-disk", "D", []string{}, "Additional disk, human readable size per entry (repeatable)")
```

?

Reply:
LGTM with this change (or without) FWIW.